code/__defines/_compile_options.dm

```diff
@@ -19,3 +19,5 @@
 	#define USING_MAP_DATUM /datum/map
 	#define MAP_OVERRIDE 1
 #endif
+

```

Comment:
Is this.... necessary?

Reply:
@DameonOwen 
honestly forgot how to revert a commit so I was like. *shrug